alf/networks/networks.py

```diff
@@ -115,3 +118,122 @@ def __init__(self, block, input_tensor_spec=None, activation=torch.relu_):
     def forward(self, x, state=()):
         y, state = self._block(x, state)
         return self._activation(x + y), state
+
+
+class TemporalStack(Network):
+    """Stack features temporally.
+
+    Suppose input_size=(), stack_size=2, pooling_size=2, the following table
```

Comment:
I feel like "stack" is not the proper word here. With pooling_size =K, a stack_size=2 is actually stacking a max of 2*K inputs. Maybe change to another word?

Reply:
If stack_size is 2, the dimension of the output is always 2 * input_dimention. And it's a stack of 2 results of pooling operations.

Reply:
My point is that the word "stack" usually simply means putting things together without changing their quantities. Since we have a pooling operation, we might want to add words like "reduce" into the name.

Reply:
Changed to TemopralPool